tests/acceptance/demo-page/file-contents-card-test.js

```diff
@@ -0,0 +1,116 @@
+import { module, test } from 'qunit';
+import { currentURL } from '@ember/test-helpers';
+import { setupApplicationTest } from 'codecrafters-frontend/tests/helpers';
+import demoPage from 'codecrafters-frontend/tests/pages/demo-page';
+
+module('Acceptance | demo page | file-contents-card', function (hooks) {
+  setupApplicationTest(hooks);
+
+  test('it lands on a correct URL', async function (assert) {
+    await demoPage.demoTabs.fileContentsCard.visit();
+    assert.strictEqual(currentURL(), '/demo/file-contents-card', 'URL is correct');
+  });
+
+  test('it renders a FileContentsCard component', async function (assert) {
+    await demoPage.demoTabs.fileContentsCard.visit();
+    assert.ok(demoPage.demoTabs.fileContentsCard.component.isPresent, 'component has rendered');
+  });
+
+  test('it allows toggling `isCollapsible` option', async function (assert) {
```

Comment:
These are fine, they're testing individual options that don't implicitly test others